src/js/components/addData/AddDataMeta.jsx

```diff
@@ -118,10 +127,42 @@ export default class AddDataMeta extends React.Component {
         }
     }
 
-    submitMetadata(){
-        this.props.updateMetaData(this.state);
-    }
-
+    submitMetadata(e){
+        let agency = this.state.agency, 
+            endDate = this.state.endDate,
+            dateType = this.state.dateType,
+            year = endDate.substr(3),
+            quarter = endDate.substr(0, 2);  
+        switch (quarter) {
+            case '12': quarter = '3';
+                // First quarter (Oct-Dec) has end-year of the previous year. Add 1 to make it the same FYE.
+                if (quarter == '3') {
+                    year = parseInt(year) + 1;
+                }
+                break;
+            case '03': quarter = '6';
+                break;
+            case '06': quarter = '9';
+                break;
+            case '09': quarter = '12';
+                break;
+            default: quarter = '0';
+            }
+        // Only make a request to check certified submission for quarterly submission. 
+        if (dateType == 'quarter') {
```

Comment:
Shouldnt the quarter conversion happen inside this if statement?
There is no need to calculate the quarter if it is not a quarterly submission